overlay/default.nix

```diff
@@ -9,7 +10,9 @@ let
 
     rustLib = callPackage ./lib { };
 
-    makePackageSet = callPackage ./make-package-set.nix;
+    makePackageSet = callPackage ./make-package-set/full.nix;
+
+    makePackageSet' = pkgs.callPackage ./make-package-set/simplified.nix;
```

Comment:
Curious: What's the simplified one used for?

Reply:
It removes a lot of boilerplate (like choosing `rustc`, `cargo`, specifying build packages, ...).